public/pages/Dashboard/utils/utils.tsx

```diff
@@ -553,27 +564,21 @@ const buildDetectorAndIdMap = (
 
 const selectLatestDetetorIds = (
```

Comment:
Small: typo - should be `selectLatestDetectorIds`

Reply:
Good catch. Thank you.